tests/queries/0_stateless/02404_rocksdb_read_only.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+# Tags: no-fasttest, no-parallel
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+ROCKSDB_DIR="/var/lib/clickhouse/store/test_rocksdb_read_only"
```

Comment:
```suggestion
ROCKSDB_DIR="$CLICKHOUSE_TMP/test_rocksdb_read_only"
```

Reply:
> 2022-08-29 15:30:02 Received exception from server (version 22.9.1):
2022-08-29 15:30:02 Code: 1001. DB::Exception: Received from localhost:9000. DB::Exception: std::__1::__fs::filesystem::filesystem_error: filesystem error: in create_directories: Permission denied ["/tmp/clickhouse-test/0_stateless/test_ho2rhg/test_rocksdb_read_only"]. (STD_EXCEPTION)
2022-08-29 15:30:02 (query: CREATE TABLE test_02404 (key UInt64, value String) Engine=EmbeddedRocksDB(0, '/tmp/clickhouse-test/0_stateless/test_ho2rhg/test_rocksdb_read_only') PRIMARY KEY(key);)

ci have no permission in `$CLICKHOUSE_TMP`, maybe we should keep `/var/lib/clickhouse/store`?

Reply:
it used in some other tests so it's strange that it doesn't work

Reply:
Okay, it is possible that from CH we cannot use the file because of ownership.
Also, I don't like modifying `/var/lib/clickhouse` folder from functional tests because the binary is shared.

What do you think about moving all these checks into an integration test?
You should have same behaviour locally when testing it out, so it would be easier to verify that it works.

Reply:
make sense. I will add a folder named `test_rocksdb_read_only`, and move the functional test into it. Also, move test_dirctory_missing_after_stop into it. Is it Ok?

Reply:
yes, it makes sense